src/Data/LifeCycle.elm

```diff
@@ -60,7 +60,7 @@ getStep label =
 
 getStepMass : Step.Label -> LifeCycle -> Mass
 getStepMass label =
-    getStep label >> Maybe.map .mass >> Maybe.withDefault (Mass.kilograms 0)
+    getStep label >> Maybe.map .inputMass >> Maybe.withDefault (Mass.kilograms 0)
```

Comment:
Maybe we shouldn't imply getting step mass is necessary the input one? We could probably replace this getter with something more general, eg.:

```elm
-- Data.LifeCycle.elm
getStepProp : Step.Label -> (Step -> a) -> LifeCycle -> Maybe a
getStepProp label prop =
    getStep label >> Maybe.map prop
```

So we could use `lifeCycle |> LifeCycle.getStepProp .inputMass` or `lifeCycle |> LifeCycle.getStepProp .outputMass` explicitely. What do you think?

Reply:
Addressed in https://github.com/MTES-MCT/wikicarbone/pull/27/commits/277585c70ba1f5ae9d17fc11886cc5603eff44ab